packages/zero-client/src/client/zero.ts

```diff
@@ -478,10 +479,19 @@ export class Zero<
     });
     const logOptions = this.#logOptions;
 
+    const enableLegacyMutators =
+      options.enableLegacyMutators ||
+      options.enableLegacyMutators === undefined;
+    const enableLegacyQueries =
+      options.enableLegacyQueries || options.enableLegacyQueries === undefined;
```

Comment:
```suggestion
```suggestion
    const {enableLegacyMutators = true, enableLegacyQueries = true} = options;
```
```